src/main/java/com/treasuredata/client/model/TDSavedQuery.java

```diff
@@ -198,6 +214,8 @@ public String toString()
                 ", result='" + result + '\'' +
                 ", nextTime='" + nextTime + '\'' +
                 ", engineVersion='" + engineVersion + '\'' +
+                ", executingUserid='" + executingUserId + '\'' +
```

Comment:
```suggestion
                ", engineVersion=" + engineVersion +
                ", executingUserId=" + executingUserId +
```

NIT: Single quotes aren't necessary.